protocol/x/clob/keeper/deleveraging.go

```diff
@@ -132,18 +132,33 @@ func (k Keeper) MaybeDeleverageSubaccount(
 
 // GetInsuranceFundBalance returns the current balance of the insurance fund (in quote quantums).
 // This calls the Bank Keeper’s GetBalance() function for the Module Address of the insurance fund.
-func (k Keeper) GetInsuranceFundBalance(
-	ctx sdk.Context,
-) (
-	balance *big.Int,
-) {
+func (k Keeper) GetInsuranceFundBalance(ctx sdk.Context, perpetualId uint32) (balance *big.Int) {
```

Comment:
nit: Update comment to note that this gets the balance for the insurance fund for a specific perpetual.